supabase/seed.sql

```diff
@@ -111,4 +111,15 @@ SELECT http_set_curlopt(''CURLOPT_TIMEOUT_MS'', ''15000'');
            ''application/json'',
            ''{}''
         )::http_request)
-    ', 'localhost', 5432, 'postgres', 'supabase_admin', 't', 'cron_everyday_web_stats');
+    ', 'localhost', 5432, 'postgres', 'supabase_admin', 't', 'cron_everyday_web_stats'),
+(15, '0 0 * * *', '
+    UPDATE app_versions
+    SET deleted = true
+    FROM apps
+    WHERE app_versions.storage_provider=''r2''
```

Comment:
any storage is ok, I don't think we need to select only r2

Reply:
Fixed in latest commit